gprofiler/main.py

```diff
@@ -177,7 +181,7 @@ def _generate_output_files(
     ) -> None:
         start_ts = get_iso8601_format_time(local_start_time)
         end_ts = get_iso8601_format_time(local_end_time)
-        base_filename = os.path.join(self._output_dir, "profile_{}".format(end_ts))
+        base_filename = os.path.join(self._output_dir, "profile_{}".format(end_ts.replace(':','-')))
```

Comment:
Why is that? Windows doesn't allow `:` in names?

Reply:
Yes, Windows doesn't allow : in names